CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(CMAKE_CXX_STANDARD 17)
 #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=thread")
 
 set(REALM_BUILD_LIB_ONLY ON)
-add_subdirectory(realm-core EXCLUDE_FROM_ALL)
```

Comment:
Why? Removing this will build all Core targets implicitly, not just the ones we explicitly depend on.

Reply:
When `EXCLUDE_FROM_ALL` was present, no core targets we depend on were being installed. Removing this solved that. Is there another more correct way to approach this?

Reply:
Oh, I didn’t realize we wanted a combined binary package of the Core static libraries and headers, together with the SDK itself. I suppose right now that’s the only way we can go about this, at least until we hide the Core headers and can build everything as a single shared library.